src/server/server_impl.go

```diff
@@ -88,6 +96,14 @@ func NewJsonHandler(svc pb.RateLimitServiceServer) func(http.ResponseWriter, *ht
 			return
 		}
 
+		// Generate trace
+		_, span := tracer.Start(context.TODO(), "NewJsonHandler Remaining Execution",
```

Comment:
Without a real context is there any real use for this?

Reply:
Resolving this conversation as it is a duplicate.